torch/_tensor.py

```diff
@@ -491,12 +491,12 @@ def backward(
             See https://github.com/pytorch/pytorch/pull/60521#issuecomment-867061780 for more details.
 
         Args:
-            gradient (Tensor or None): Gradient w.r.t. the
-                tensor. If it is a tensor, it will be automatically converted
+            gradient (Tensor or None, optional): The gradient of the function
+                being differentiated w.r.t. ``self``.
+                If it is a tensor, it will be automatically converted
                 to a Tensor that does not require grad unless ``create_graph`` is True.
-                None values can be specified for scalar Tensors or ones that
-                don't require grad. If a None value would be acceptable then
-                this argument is optional.
+                None values can be specified for scalar tensors or tensors that
```

Comment:
Since self must require grad, we can further simplify this to:
"This argument can be omitted if self is a scalar".